notice/overrides.json

```diff
@@ -1,3 +1,4 @@
+{"name": "github.com/elastic/e2e-testing/cli", "licenceType": "Elastic"}
```

Comment:
Is there a way to exclude this file from pre-commit JSON checks? It's not a valid JSON file, although it's a valid file for the license tool

Reply:
yes, you can use the exclude filter:

```yaml
    -   id: check-json
        exclude: ^notice/overrides.json
```

See https://pre-commit.com/#pre-commit-configyaml---top-level as a reference :)